routers/static_filter.go

```diff
@@ -95,6 +96,41 @@ func fastAutoSignin(ctx *context.Context) (string, error) {
 	return res, nil
 }
 
+func InsertOrganizationTheme(ctx *context.Context, urlPath string) {
```

Comment:
Rename to `appendThemeCookie`, put to a new file: `theme_filter.go`

return error